src/platform-implementation-js/dom-driver/gmail/views/gmail-compose-view/setupLinkPopovers.ts

```diff
@@ -30,6 +30,13 @@ class LinkPopOver extends SafeEventEmitter {
     );
   }
 
+  getTextInputElement() {
+    // this is for the new gmail UI
```

Comment:
oh sorry i added the comment in wrong PR: https://github.com/InboxSDK/InboxSDK/pull/1239/files#r1924283251, meant to be here

Reply:
with all current changes, we have site logging and new class, should be able to find out who has the new UI. resolving this comment